### PR TITLE
L is measured in blocks for block ciphers.

### PR DESCRIPTION
**Pull request by chris-wood**

### CODE REVIEW

**Comment by martinthomson** [comment]
I think that this is the right change, but we need to make sure that we change the heading of the column to "message size".  I need to check the math still though.

**Review by martinthomson — APPROVED** [review]
Just one thing to update: the title of the column needs to be "message size" not "l or s" because we really are counting in bytes.  That suggests that you might want a different variable name for what is passed in.

File: index.html [hunk 1]
```diff
@@ -159,24 +160,28 @@
...
 			ilimit: function(p, l, q) {
-				return ccm_ilimit("AEAD_AES_128_CCM_8", l);
+				let l_in_blocks = l / 16;
+				return ccm_ilimit("AEAD_AES_128_CCM_8", l_in_blocks);
 			}
 		},
 	];
 	let sizes = [];
-	for (i = 0; i <= 16; i += 2) {
+	for (i = 4; i <= 16; i += 2) {
```
Comment (martinthomson) [inline index.html:184]: Oh, I thought that 0 might be interesting for cases where you have a tag on every block.  That might happen for something like NTP that has very small messages, but maybe they aren't that small and it's rare for a use case to have a maximum message size that small.

Comment (chris-wood) [reply index.html]: The CCM confidentiality bound results in infinity if l = 0 (we divide by l^2). I left it in just for kicks, and added back the [1,2,4,8] values.

**Pull request merged**